src/store/modules/auth-module.js

```diff
@@ -0,0 +1,55 @@
+import getCookie from '@/helpers/get-cookie';
+import setCookie from '@/helpers/set-cookie.js'
+export default {
+    namespaced: true,
+    state: {
+        token: null,
+        user: {},
+        password: "",
+    },
+    getters: {
+        isLogged(state) {
+            return Object.keys(state.user).length > 0 && state.token !== null;
+        },
+    },
+    mutations: {
+        setToken(state, token) {
+            state.token = token;
+        },
+        setUser(state, user) {
+            state.user = user;
+        },
+        setPassword(state, password) {
+            state.password = password;
+        }
+    },
+    actions: {
+        register({
+            commit,
+            state
+        }, registerData) {
+            commit("setToken", registerData.token)
+            commit("setUser", registerData.user)
+            setCookie("token", state.token, 1000000)
+            setCookie("user", JSON.stringify(state.user), 1000000)
+        },
+        autoLogin({
+            commit
+        }) {
+            const cookieToken = getCookie("token")
+            const cookieUser = JSON.parse(getCookie("user"))
+            if (cookieToken) {
+                commit("setToken", cookieToken)
+            }
+            if (cookieUser) {
+                commit("setUser", cookieUser)
+            }
+        },
+        login({
```

Comment:
Ta akcja jest niepoprawnie zaimplementowana. Nie odbiera payloadu i nie zapisuje danych otrzymanych z REST API. Wszystko co robi to ustawia cookiesy ze stanu. Co w przypadku jeśli ich nie będzie?

Reply:
Co w takim przypadku miss być payloadem ? Nie rozumiem koncepcji